File: clase-08-19/main.cpp
#include "tads/LinkedQueue.cpp"
#include <iostream>

int main()
{
  int totalElements;
  std::cin >> totalElements;
  Queue<int> *queue = new LinkedQueue<int>;

  for (int i = 0; i < totalElements; i++)
  {
    int element;
    std::cin >> element;

    queue->push(element);
  }

  while (!queue->isEmpty())
  {
    std::cout << queue->top() << std::endl;
    queue->pop();
  }
}

File: clase-08-19/Main.java
import java.util.Scanner;

import tads.LinkedQueue;
import tads.Queue;

public class Main {
  public static void main(String[] args) {
    int elements;

    Queue<Integer> queue = new LinkedQueue<Integer>();

    Scanner in = new Scanner(System.in);
    elements = in.nextInt();

    for (int i = 0; i < elements; i++) {
      int data = in.nextInt();

      queue.push(data);
    }

    while (!queue.isEmpty()) {
      System.out.println(queue.top());
      queue.pop();
    }
  }
}
